neo-cli/CLI/ConsolePercent.cs

```diff
@@ -1,3 +1,13 @@
+// Copyright (C) 2016-2021 The Neo Project.
+// 
+// The Neo.CLI is free software distributed under the MIT software 
```

Comment:
```suggestion
// The neo-cli is free software distributed under the MIT software 
```

It should be the same as the project name.